doc/distrib/NodeHelpFiles/_review/Revit.Elements.FamilyInstance.FacingOrientation.md

```diff
@@ -0,0 +1,8 @@
+## In Depth
+`FamilyInstance.Location` returns a vector describing the orientation of a family instance. This can be used to determine how much an instance has been rotated.
```

Comment:
Node name is incorrect. Replace FamilyInstance.Location with FamilyInstance.FacingOrientation.

Reply:
Whoopsies. Fixed now.